ansible/playbooks/service/grafana.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Manage Grafana
+  hosts: [ 'debops_service_grafana', 'debops_grafana' ]
```

Comment:
Don't use the `debops_grafana` group scheme, they were removed before the `v2.0.0` release.

Reply:
removed